tutorials/XGI in 5 minutes.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>nice! i think we should have only integer xticks, and also enforce that the bins are centered on the integers. also maybe add "# occurrences" for the ylabel</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>